### PR TITLE
Ekiga 4

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/desktop/ekiga/Makefile [hunk 1]
```diff
@@ -12,25 +12,33 @@
...
 include $(WS_TOP)/make-rules/ips.mk
 
 PATH=/usr/gnu/bin:/usr/bin:/usr/perl5/bin
 
+COMPONENT_PREP_ACTION = ( cd $(@D) && \
+          libtoolize --force && \
+          aclocal -I. && \
+          automake -a -f -c &&\
+          autoconf )
+
```
Comment (pyhalov) [inline components/desktop/ekiga/Makefile]: Please, drop depend.mk and add REQUIRED_PACKAGES

**Comment by alarcher** [comment]
Tested with webcam and calls through voip provider.

File: components/ekiga/patches/ekiga-02-openldap-location.patch [hunk 2]
```diff
@@ -1,51 +0,0 @@
---- ekiga-3.2.5/configure.ac.orig	2009-07-07 03:09:34.000000000 +0800
```
Comment (pyhalov) [inline components/ekiga/patches/ekiga-02-openldap-location.patch:1]: Does it detect openldap automagically now?

Comment (alarcher) [reply components/ekiga/patches/ekiga-02-openldap-location.patch:1]: LDAP was already disabled in the component so I did not test it.

**Comment by pyhalov** [comment]
So,  can we merge it (with necessary libraries)?  Or somehow call for testing?

**Comment by alarcher** [comment]
The last thing is I do not know what ekiga-06-t140.patch fixed and if it is any relevant.
With my use of the past days I see no regression so we could merge it and then people could test until October.

**Comment by pyhalov** [comment]
Evidently, added T.140 support, https://en.wikipedia.org/wiki/Text_over_IP . Ekiga still misses it. But I don't know how widely it's used (just have read about it for the first time).

**Comment by alarcher** [comment]
I should have worded it differently: I do not know how relevant it is but I can update the patch if you like.

**Comment by pyhalov** [comment]
I don't care much about it, as I've just heard about it for the first time. If you update patch, you should somehow test it. Or find user of this feature on ML/IRC.

**Comment by alarcher** [comment]
I looked at it and while most changes are easy, interfaces in opal have changed so it needs some work.
I would say merge it.

**Pull request merged**